content/doc/upgrade/from-gogs.en-us.md

```diff
@@ -15,12 +15,37 @@ menu:
 
 # Upgrade from Gogs
 
-If you are running Gogs which version is below 0.9.99, you can upgrade Gogs to Gitea smoothly. There are some steps to do below:
+If you are running Gogs version 0.9.113 or older, you can upgrade Gogs to Gitea smoothly. There are some steps to do below. Make sure to complete as the gitea user in ~:
 
 * Stop Gogs
-* Copy Gogs `custom/conf/app.ini` to Gitea.
-* Copy Gogs `conf/` to `options/` for Gitea.
-* If you have more custom content on `custom/` folder, like templates or localization files, you should carrefully merge manually because these files changed on Gitea.
-* Copy `data/` folder from Gogs to Gitea. It contains issue attachments and avatars.
```

Comment:
lunny, I had `cp -r gogs gitea` that should copy the data/ folder from gogs to gitea. Do you want me to break that out as a separate line?